README.md

```diff
@@ -1,13 +1,15 @@
 <div align="center">
 
-<h1> Mia-Platform Community </h1>
+# Mia-Platform Community
 
-<strong>Welcome to the [Mia-Platform][mia-website] Community!</strong>
+**Welcome to the [Mia-Platform][mia-website] Community!**
 
 Head to the [Discussion Section][mia-community] to join!
 
-</div>
+If you wish to request the addition of a new component to Mia Platform marketplace visit the [Issues Section][mia-issues] and choose the **Marketplace contribution** template.
```

Comment:
```suggestion
If you wish to request the addition of a new component to Mia-Platform marketplace visit the [Issues Section][mia-issues] and choose the **Marketplace contribution** template.
```

Reply:
Instead of telling to select the template you can use the link with the query parameter `?template=marketplace-contribution.md` and get the user to the correct template immediately

Reply:
Applied both suggestions